### PR TITLE
Update local file help text

### PR DESCRIPTION
**Pull request by amacneil**
**User-Facing Changes**
Update text below `Open local file` in welcome dialog.

image

**Description**
We had some feedback from users that they were confused whether `Open local file` would upload the file to Foxglove. This is an attempt at improving the text to reassure folks.

FG-989

### CODE REVIEW

**Comment by amacneil** [comment]
Looking for feedback on whether people like this wording, or have better suggestions.

File: packages/studio-base/src/components/OpenDialog/Start.tsx [hunk 1]
```diff
@@ -410,21 +409,17 @@ function SidebarItems(props: { onSelectView: (newValue: OpenDialogViews) => void
...
-    const formatter = new Intl.ListFormat("en-US", { style: "long" });
-    const supportedLocalFiles = formatter.format(
-      Array.from(new Set(supportedLocalFileExtensions)).sort(),
-    );
     return [
       {
         key: "open-local-file",
         text: "Open local file",
-        secondaryText: `Supports ${supportedLocalFiles} files.`,
+        secondaryText: "Visualize directly from your local filesystem.",
```
Comment (esthersweon) [inline packages/studio-base/src/components/OpenDialog/Start.tsx:422]:
```suggestion
        secondaryText: "Visualize data directly from your local filesystem.",
```

**Review by esthersweon — APPROVED** [review]

File: packages/studio-base/src/components/OpenDialog/Start.tsx (hunk 1 above)
Comment (esthersweon) [inline packages/studio-base/src/components/OpenDialog/Start.tsx:422]:
```suggestion
        secondaryText: "Visualize data directly from your local filesystem.",
```

I think having in-app documentation about what the supported file formats are is still useful. Could we combine both?

**Review by ocin-rye — APPROVED** [review]
Looks good. It should reduce some of the confusion experienced by users.

**Comment by defunctzombie** [comment]
Bowing out of the review here. I am confident the remaining wordsmiths will produce a good result.

**Comment by amacneil** [comment]
> I think having in-app documentation about what the supported file formats are is still useful. Could we combine both (e.g. "Visualize ${supportedLocalFiles} files directly from your local filesystem")?

@esthersweon we do link to the docs from the welcome dialog, and the "open file" picker will already highlight supported file types. Do you think we need more than that?

**Pull request merged**